src/SFA.DAS.EmployerUsers.Web/Views/Account/Register.cshtml

```diff
@@ -29,31 +33,31 @@
         <label for="Email">
             <span class="form-label-bold">Email</span>
             <span class="form-hint">We'll send you an email to confirm your identity.</span>
-            <span class="error-message" id="error-email">@Html.Raw(Model.Data.EmailError.Replace("|",""))</span>
+            <span class="error-message" id="error-email">@Html.Raw(Model.Data.EmailError.Replace("|", ""))</span>
         </label>
         <input class="form-control" id="Email" name="Email" type="text" value="@Html.Encode(Model.Data.Email)" required="required">
     </div>
     <div class="form-group @(!string.IsNullOrEmpty(Model.Data.PasswordError) ? "error" : "") ">
         <label for="Password">
             <span class="form-label-bold">Create new password</span>
             <span class="form-hint">Password requires upper and lowercase letters, a number and at least 8 characters.</span>
-            <span class="error-message" id="error-password">@Model.Data.PasswordError</span>     
-        </label> 
-        <input class="form-control" id="Password" type="password" name="Password" required="required">        
+            <span class="error-message" id="error-password">@Model.Data.PasswordError</span>
+        </label>
+        <input class="form-control" id="Password" type="password" name="Password" required="required">

```

Comment:
Not sure where the 'noise' of this diff comes from, but it doesn't look like it is actually changing anything.